test/unit/errorResponseInterceptor.test.js

```diff
@@ -362,58 +351,49 @@ describe('test errorResponseInterceptor', () => {
     }
   });
 
-  it('test_added', async () => {
-    const expectedNumberOfResponseInterceptors = numberOfBaseInterceptors + 1;
+  /**
+   * Assert the error interceptors are present and correctly ordered.
+   *
+   * @returns the expected number of interceptors
+   */
+  function assertInterceptors() {
     assert.strictEqual(
       service.requestWrapperInstance.axiosInstance.interceptors.response
         .handlers.length,
       expectedNumberOfResponseInterceptors
     );
-    // check whether errorResponseInterceptor is set as a new interceptor for rejected responses
-    const actualErrorResponseInterceptor =
+    // check the 2 error response interceptors are set for rejected responses
+    // errorResponseStreamConverter and
+    // errorResponseInterceptor
+    // and ensure the stream converter is first
+    const actualFirstRejectionInterceptor =
       service.requestWrapperInstance.axiosInstance.interceptors.response
-        .handlers[expectedNumberOfResponseInterceptors - 1].rejected;
-    assert.deepStrictEqual(
-      Object.getPrototypeOf(actualErrorResponseInterceptor),
-      Object.getPrototypeOf(errorResponseInterceptor)
-    );
-  });
-  it('test_added_via_configureService', async () => {
-    // Before configureService:
-    let expectedNumberOfResponseInterceptors = numberOfBaseInterceptors + 1;
-    assert.strictEqual(
+        .handlers[firstErrorInterceptorIndex].rejected;
+    const actualSecondRejectionInterceptor =
       service.requestWrapperInstance.axiosInstance.interceptors.response
-        .handlers.length,
-      expectedNumberOfResponseInterceptors
+        .handlers[secondErrorInterceptorIndex].rejected;
+    assert.strictEqual(
+      actualFirstRejectionInterceptor,
+      errorResponseStreamConverter
     );
-    // check whether errorResponseInterceptor is set as a new interceptor for rejected responses
-    let actualErrorResponseInterceptor =
-      service.requestWrapperInstance.axiosInstance.interceptors.response
-        .handlers[expectedNumberOfResponseInterceptors - 1].rejected;
-    assert.deepStrictEqual(
-      Object.getPrototypeOf(actualErrorResponseInterceptor),
-      Object.getPrototypeOf(errorResponseInterceptor)
+    assert.strictEqual(
+      actualSecondRejectionInterceptor,
+      errorResponseInterceptor
     );
-    // get number of service response interceptors before configureService
-    expectedNumberOfResponseInterceptors =
-      service.requestWrapperInstance.axiosInstance.interceptors.response
-        .handlers.length; // after configureService we should get the same number of interceptors
+    return expectedNumberOfResponseInterceptors;
```

Comment:
Do we need this return value?

Reply:
No, I used it briefly then changed my mind and forgot to remove it.
Cleaned up in e2f1e377